### ISSUE TITLE
Is it possible to add and open relative paths?

### ISSUE THREAD

**Issue by sahil48** [fabiospampinato/vscode-todo-plus#265]
![image](https://user-images.githubusercontent.com/6100774/82208789-26d4e380-98da-11ea-95f6-26398ad29b3d.png)

When you enter a link in a markdown document, it is possible to open the links straight in VS-Code, including relative paths. It enables me to quickly move between files in the same folder. Is there a format for these links that Todo-Plus would recognize?

**Comment by fabiospampinato** [comment]
Absolute paths prefixed with `file://` should be recognized. I think perhaps support for relative paths could be implemented at the extension level (although I see it is underlined in the screenshot, isn't it recognized already? Perhaps you need to associate taskpaper files with vscode?)

**Comment by sahil48** [comment]
The file in the picture is a markdown file. I provided it to illustrate what a clickable link looks like in VSCode. It is not recognized with .taskpaper documents. I think most likely, this would be best done at the extension level, because it seems the links vary by syntax highlighting. For now, Alt-P does navigate to the file, but it would be nice to have it clickable for better discoverability.

**Comment by fabiospampinato** [comment]
> this would be best done at the extension level, because it seems the links vary by syntax

Good point 🤔 Todo+ doesn't really have a syntax for links though other than general `file://` urls.

**Comment by vfonic** [comment]
Relative file paths also seem to work in this format:

file://./config/initializers/20_apps_configuration.rb => links to `config/initializers/20_apps_configuration.rb`

I also tried to add line number but none of the below worked:

- file://./config/initializers/20_apps_configuration.rb:11
- file://./config/initializers/20_apps_configuration.rb#L11
- file://./config/initializers/20_apps_configuration.rb:11:1
- file://./config/initializers/20_apps_configuration.rb?line=11
- file://./config/initializers/20_apps_configuration.rb#11
- file://./config/initializers/20_apps_configuration.rb?start=11
- file://./config/initializers/20_apps_configuration.rb(11)
